webapp/src/main/java/org/iqu/webapp/events/management/RetrieveAuthorsService.java

```diff
@@ -1,39 +1,39 @@
 package org.iqu.webapp.events.management;
 
+import java.util.Set;
+
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 
-import org.iqu.webapp.entities.Authors;
-import org.iqu.webapp.entities.ErrorMessage;
+import org.iqu.slaveservices.entities.ErrorMessage;
+import org.iqu.webapp.factory.ServiceFactory;
+
+import orq.iqu.slaveservices.events.EventsService;
 
 /**
- * RetrieveCategoriesService - Class that implements retrieve authors service.
+ * RetrieveAuthorsService - Class that implements retrieve authors service.
  * 
  * @author Alex Dragomir
  *
  */
```

Comment:
if this service is defined for the /authors path, it is unclear whether the authors are for news or for events. The paths should suggest a hierarchical service structure. An endpoint should be defined for news, another one for events and so on. And under those endpoints the corresponding /news/authors or events/authors methods can be implemented.